### DISCUSSION TITLE
brew upgrade - URL failed for libnghttp2

### DISCUSSION THREAD

**Discussion by ddspell** [Homebrew/discussions#3952] [Everyday usage] [answered]
Not sure, but maybe there's a permission issue.
If I try to reach the URL, I get "UNAUTHORIZED".
{"errors":[{"code":"UNAUTHORIZED","message":"authentication required"}]}

Running on an Apple M1 Pro MacBook Pro running macOS Monterey 12.6.1.
laptop ~ % brew --version
Homebrew 3.6.12
Homebrew/homebrew-core (git revision 221eab6c3ab; last commit 2022-11-22)
Homebrew/homebrew-cask (git revision 84a45f6e4d; last commit 2022-11-22)

**Example**
laptop ~ % brew update
Updated 2 taps (homebrew/core and homebrew/cask).
==> Outdated Formulae
libnghttp2

You have 1 outdated formula installed.
You can upgrade it with brew upgrade
or list it with brew outdated.

laptop ~ % brew outdated
libnghttp2 (1.50.0) < 1.51.0

laptop ~ % brew upgrade
==> Upgrading 1 outdated package:
libnghttp2 1.50.0 -> 1.51.0
==> Downloading https://ghcr.io/v2/homebrew/core/libnghttp2/manifests/1.51.0
curl: (22) The requested URL returned error: 404

Error: libnghttp2: Failed to download resource "libnghttp2_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/libnghttp2/manifests/1.51.0

**Comment by SMillerDev** [comment, marked as answer]
See https://github.com/Homebrew/homebrew-core/pull/115661

**Reply by fxcoudert** [reply]
The issue is now fixed. Run `brew up` before installing/reinstalling/upgrading, and you'll be good to go.

**Reply by qiuyaofan** [reply]
brew update
